src/traces/image/attributes.js

```diff
@@ -51,6 +51,18 @@ module.exports = extendFlat({
             'otherwise it defaults to `rgb`.'
         ].join(' ')
     },
+    interpolate: {
+        valType: 'enumerated',
+        values: ['nearest', 'linear'],
```

Comment:
`linear` or `bilinear`?

I should also mention that the CSS spec does not seem to enforce an interpolation method and thus the result might be browser dependent (https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering#Values)

![Screenshot_2020-12-18_11-16-50](https://user-images.githubusercontent.com/301546/102636456-aa7dd980-4122-11eb-8fe0-04bc52f96cf6.png)

Reply:
> the CSS spec does not seem to enforce an interpolation method

That's a good point. Browsers could use e.g. cubic or Lanczos interpolation just as well.

Reply:
Or nearest.